scripts/globals/mobskills/Serpentine_Tail.lua

```diff
@@ -13,21 +13,21 @@ require("scripts/globals/monstertpmoves");
 ---------------------------------------------
 function onMobSkillCheck(target,mob,skill)
 
-  if(mob:getFamily() == 316) then
-    local mobSkin = mob:getModelId();
+    if (mob:getFamily() == 316) then
+        local mobSkin = mob:getModelId();
 
-    if (mobSkin == 1796) then
+        if (mobSkin == 1796) then
+            return 0;
+        else
+            return 1;
+        end
+    end
+
+    if (mob:AnimationSub() < 2 and target:isBehind(mob, 48) == true and mob:getFamily() == 313) then -- Tinnin
```

Comment:
To be honest I don't like all these conditions. I feel like it's better to make multiple mob skill lists and change the mobs skill list when it changes states. You can do it from lua like so `mob:setMobMod(MOB_SKILL_LIST, {listId})`